browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -95,6 +96,24 @@ class MarkdownNoteDetail extends React.Component {
     return title
   }
 
+  getPercentageOfCompleteTodo (value) {
```

Comment:
Thank you for commenting ! Surely it is difficult to understand it mean. I'll fix it.